src/vmm/src/vm.rs

```diff
@@ -0,0 +1,105 @@
+use crate::vcpu::{self, Vcpu};
+use kvm_bindings::{kvm_pit_config, kvm_userspace_memory_region, KVM_PIT_SPEAKER_DUMMY};
+use kvm_ioctls::{Kvm, VmFd};
+use std::sync::Arc;
```

Comment:
nit: reorder these `use`s to separate `std` from internal crates and everything from extern crates

Reply:
Done.